src/DBConnection2.cpp

```diff
@@ -0,0 +1,20 @@
+#include <iostream>
```

Comment:
@Morheit , I see no iostream usage

Reply:
@malirod , oops, seems I forgot to delete it. Unused `<iostream>` deleted